src/platforms/dotnet/guides/xamarin/common/troubleshooting.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title:  Troubleshooting
+sidebar_order: 100
+description: "Solutions to common issues"
+---
+
```

Comment:
suggest adding an intro here:
"If you need help solving issues with your Sentry's Xamarin SDK, you can read the edge cases documented here. If you need additional help, you can view our forums, and customers on a paid plan may also contact support."

Reply:
sounds great :D